src/7-to-8/major-changes/before-you-start.rst

```diff
@@ -0,0 +1,89 @@
+.. _AutoConfigUpgrades:
+
+Cylc Configuration Upgrader
+===========================
+
+.. admonition:: Does This Change Affect Me?
+   :class: tip
+
+   This change may affect you:
+
+   - If you are working with workflows written before Cylc 7.
+   - If you hope to use Cylc 9.
+
+   You have been affected by this change if you see a message such as:
+
+   .. code-block::
+
+      > cylc validate my_workflow
+      IllegalItemError: reallyoldconfig
+
+
+Overview
+--------
+
+Cylc has a built in configuration upgrader. Cylc can upgrade Cylc 7
+workflows to Cylc 8 workflows. Cylc cannot upgrade Cylc 6 or earlier
```

Comment:
This might give the impression it will upgrade the file itself?
```suggestion
Cylc has a built in configuration upgrader. Cylc 8 can upgrade Cylc 7
workflows at runtime. Cylc cannot upgrade Cylc 6 or earlier
```